components/03-organisms/galleries/media-grid/yds-media-grid.twig

```diff
@@ -0,0 +1,34 @@
+{#
+ # Available Variables:
+ # - media_grid__heading (optional)
+ # - media_grid__variation: basic (default), interactive
+ #
+ # Available Blocks:
+ # - media_grid__items
+ #}
+
+{% set media_grid__base_class = 'media-grid' %}
+
+{% set media_grid__attributes = {
+  'data-component-width': 'feature',
+  'data-media-grid-variation': media_grid__variation|default('basic'),
+} %}
+
+<div {{ bem(media_grid__base_class) }} {{ add_attributes(media_grid__attributes) }}>
```

Comment:
`bem()` should be moved up into `media_grid__attributes`

Reply:
Done